### DISCUSSION TITLE
Missing option for centered UI

### DISCUSSION THREAD

**Discussion by antyone** [C1rdec/PoeLurker#404] [Ideas]
If you use centered UI option like me, the UI is actually in the way if you look at the screen I posted with debug-mode on

![image](https://i.imgur.com/1Ejwpht.png)

I suggest an option for centered UI or an option to move that part of UI to wherever the user wants, if possible

**Comment by C1rdec** [comment]
Wont be supported. Please look into closed issues before opening discussions:

https://github.com/C1rdec/Poe-Lurker/pull/369#issuecomment-1379786449

**Reply by antyone** [reply]
I dont use github so i didnt know how to look through anything, sorry

Im also not sure what the issue is, yea the game UI moves on opened stash but at least the addon UI in the gif you posted is centered and out of the way of game UI, that is still better than what you have right now, with no option to move it out of the way and force you to play without centered-UI option in game.

The most important aspect of it is to have the addon UI out of the way during gameplay, if the UI isn't perfect on opened stash, I would much rather take that over UI being over my flasks.

Maybe im wrong about the problem or I'm missing something but thats just my 2 cents
